crates/core/src/subscription/module_subscription_manager.rs

```diff
@@ -241,7 +246,21 @@ impl SubscriptionManager {
                 );
             drop(span);
 
-            let _span = tracing::info_span!("eval_send").entered();
+            if !eval.contains_key(&event.caller_identity) {
```

Comment:
Do we want to send it to all clients with the same identity? I think probably we only want to send it to the (identity, address) pair, right?

Reply:
Good question. This will *only* send to the original caller, cause I pass the caller connection client back from the reducer call, so we won't send to all of the clients with the same identity, but there is likely a bug in the `if` statement itself. If there are two clients connected with the same identity and if only the client that is *not* making the call is subscribed, this will return false and the caller will not get the update.

Reply:
In order to fix the bug in this PR I have to fix the existing bug, so I'll submit the bugfix

Reply:
So the bug was already fixed yesterday in #1121, I updated this PR to reflect the changes

Reply:
Quite a saga! I'm glad it uncovered other issues though 🙏